src/docmanager/index.ts

```diff
@@ -399,7 +442,10 @@ class DocumentManager {
   }
 
   private _data: { [key: string]: Private.IDocumentData } = Object.create(null);
-  private _fileTypes: { [key: string ]: string[] } = Object.create(null);
+  private _modelFactories: { [key: string]: IModelFactory } = Object.create(null);
```

Comment:
Grr. Github doesn't auto-update comments here, so if you don't refresh, you don't see other people's comments.